.github/workflows/setup.yml

```
@@ -1,41 +1,63 @@
 # This is a basic workflow to help you get started with Actions
 
-name: CI
+name: Setup
 
-# Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the "main" branch
   push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+    branches: [ "open_pdks_testing" ]
+  # pull_request:
```

```suggestion
  pull_request:
```
Why are you limiting it to just this branch? If you don't add a `branches` field, it will default to all branches

---

Oh sorry I didn't revert the changes I made. I did this so it only ran one workflow when testing, as otherwise it would run multiple